alumnium/caching.py

```diff
@@ -0,0 +1,62 @@
+from os import getcwd
```

Comment:
@p0deje could you rename the file to `cache.py` to be consistent with the class name?